src/cloudburst/providers/aws/rds_instance.py

```diff
@@ -0,0 +1,151 @@
+"""
+RDS Resource for AWS provider
+"""
+
+from cloudburst.providers.base import AWSService, opcode
+from cloudburst.utils.resource_factory import aws_factory
+from cloudburst.utils.errors import NoResourcesError
+from cloudburst.utils.utils import aws_paginator
+from cloudburst.utils.shared_vars import AWS_REGIONS
+
+import boto3
+
+
+class RDSInstance(AWSService):
+    """
+    Service object for an RDS Instance
+     
+    Args:
+        session (boto3.session): A boto3 session to use for fetching resources
+    """
+
+    def __init__(self, session):
+        self._session = session
+        self._resources = []
+        self._client_map = {}
+
+    @opcode
+    def STOP_INSTANCE(self, resource):
+        """
+        Terminate an RDS instance, registered as an OPCODE
+         
+        Args:
+            resource (Resource): The resource object to terminate
+         
+        Returns:
+            N/A
+         
+        Raises:
+            N/A
+        """
+        if resource.Engine == 'aurora':
+            pass  # Aurora instances will be handled through the RDSCluster service class
+        else:
+            self.client(resource.Region).stop_db_instance(
+                DBInstanceIdentifier=resource.DBInstanceIdentifier
+            )
+
+    @property
+    def supported_regions(self):
+        """
+        Returns:
+            aws_regions (dict of regions): Return the dictionary of supported regions
+                                           formatted as 'region identifier': 'human readable
+                                           name'. To be used while fetching and modifying
+                                           resources.
+        """
+        return AWS_REGIONS
```

Comment:
Is RDS supported in all of the AWS regions?

Reply:
I believe so. I copied your code from ec2_instance.py for AWS multi-region support. Like EC2Instance, RDSInstance keeps a client_map dictionary mapping AWS region names from AWS_REGIONS to boto3 clients instantiated for each region. self.fetch_all() iterates through all regions in AWS_REGIONS.

Let me know if that answers your questions.